### ISSUE TITLE
Strange behaviour on the train and test accuracies

### ISSUE THREAD

**Issue by lcit** [dimatura/voxnet#6]
I'm running the test example you propose using:

python train.py config/shapenet10.py shapenet10_train.tar

I see that the train accuracy goes up to 100% already from the 3rd iteration then it start to oscillate between 0% and 100% which is quite strange for a neural network (the loss seems coherent but it oscillate too a bit). Then if I run the test (I have set 'checkpoint_every_nth' : 1) using:

python test.py config/shapenet10.py shapenet10_test.tar

I get a very low accuracy of less than 10% ( or at least for the first 4 epochs). Is this normal? I have not modified the code.

Thanks

**Comment by lcit** [comment]
I started again the test and it works now.
